2_process.R

```diff
@@ -62,6 +63,17 @@ p2_targets_list <- list(
     p2_site_list_nontidal_csv,
     create_site_list_nontidal(p2_wqp_SC_filtered,p1_nwis_sites,p1_daily_data,p1_inst_data,
                               hucs=drb_huc8s,crs_out="NAD83",p2_sites_w_segs,"2_process/out/DRB_SC_sitelist_nontidal.csv"),
-    format = "file")
+    format = "file"),
+  
+  # Return natural baseflow estimates for each PRMS segment
+  tar_target(
+    p2_natural_baseflow,
+    munge_natural_baseflow(baseflow_pred_files = p1_natural_baseflow_csv,
+                           segs_w_comids = p2_prms_nhdv2_xwalk %>% 
+                             select(PRMS_segid,comid_down) %>% 
+                             rename('COMID' = 'comid_down'),
```

Comment:
Do you expect that we'll need this renamed column for other functions?

Reply:
I don't really expect to use the column labeled 'COMID' again. In the crosswalk table I like keeping the column name `comid_down` because it helps signify that that column represents the COMID at the downstream end of a given PRMS segment. 

I renamed the column here rather than using `comid_down` in the function to 1) make a join step within the function _slightly_ easier to read/interpret, and 2) to perhaps give a user more latitude in choosing the COMIDs of interest beyond having a specific column called `comid_down`. If the renaming step makes the target more difficult to read, I'd opt for just requiring a column named `comid_down` for the function `munge_natural_baseflow`.

Reply:
Okay, thanks. I think it's easier the way you have it; I was just curious if the name change should be permanent